scripts/osm2pgsql-replication

```diff
@@ -115,6 +134,84 @@ def update_replication_state(conn, table, seq, date):
 
     conn.commit()
 
+def status(conn, args):
+    """\
+    Print information about the current replication status
+    """
+
+    results = {'server': {}, 'local': {}}
+
+    with conn.cursor() as cur:
+        cur.execute('SELECT * FROM pg_tables where tablename = %s', (args.table, ))
+        if cur.rowcount < 1:
+            results['error'] = {
+                    'text': {'en': "Cannot find replication status table. Run 'osm2pgsql-replication init' first."},
+                    'code': (1 << 0)
+                    }
+        else:
+            cur.execute('SELECT * FROM "{}"'.format(args.table))
+            if cur.rowcount != 1:
+                results['error'] = {
+                        'text': {'en': "Updates not set up correctly. Run 'osm2pgsql-updates init' first."},
+                        'code': (1 << 1)
+                        }
+            else:
+
+                base_url, db_seq, db_ts = cur.fetchone()
+                db_ts = db_ts.astimezone(dt.timezone.utc)
+                results['server']['base_url'] = base_url
+                results['local']['sequence'] = db_seq
+                results['local']['timestamp'] = db_ts.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+                repl = ReplicationServer(base_url)
+                server_seq, server_ts = repl.get_state_info()
+                server_ts = server_ts.astimezone(dt.timezone.utc)
+
+                results['server']['sequence'] = server_seq
+
+                results['server']['timestamp'] = server_ts.strftime("%Y-%m-%dT%H:%M:%SZ")
+                server_age = dt.datetime.now(dt.timezone.utc) - server_ts
+                results['server']['age'] = {}
+                results['server']['age']['seconds'] = int(server_age.total_seconds())
+                results['server']['age']['text'] = {'en': pretty_format_timedelta(server_age) }
+
+                results['local']['age'] = {}
+                local_age = dt.datetime.now(dt.timezone.utc) - db_ts
```

Comment:
`now()` should be called only once or you might end up with odd discrepancies for the age.